pkg/controller/apicurioregistry/svc/resources/resource_cache.go

```diff
@@ -46,8 +46,6 @@ type resourceCacheEntry struct {
 	isPatched     bool
 }
 
-var _ ResourceCacheEntry = &resourceCacheEntry{}
```

Comment:
This is a check to make sure interface is implemented.

Reply:
bring this back